packages/storybook/src/Table/Table.stories.tsx

```diff
@@ -149,18 +150,29 @@ export function Initial() {
   })
 
   const labelRef = React.useRef<HTMLHeadingElement>(null)
+  const [hasFixedLayout, setHasFixedLayout] = React.useState(true)
 
   return (
     <>
       <h1 ref={labelRef} id="Testing">
         Table example
       </h1>
 
-      <Table ref={node => register(node, { describedby: labelRef })}>
+      <Table
+        ref={node => register(node, { describedby: labelRef })}
+        sx={{
+          tableLayout: hasFixedLayout ? 'fixed' : 'auto',
+          width: hasFixedLayout ? 'auto' : '100%',
+        }}
+      >
```

Comment:
@alexanderchan I think we should move this logic into the hook some how so that all a developer has to do is set the `width` on the column and everything else will be taken care of.

Reply:
I've added it to the getTableProps and then the sx could be merged in with any user preferences if they need to.